aptos-move/framework/aptos-framework/sources/governed_gas_pool.move

```diff
@@ -71,6 +85,11 @@ module aptos_framework::governed_gas_pool {
         move_to(aptos_framework, GovernedGasPool{
             signer_capability: governed_gas_pool_signer_cap,
         });
+
+        move_to(aptos_framework, GovernedGasPoolV2{
```

Comment:
this is only called during genesis. You need to a separate function to initialize this resource. Otherwise, deposit and withdraw would break